vagrant_support/checksetup_answers.j2

```diff
@@ -0,0 +1,37 @@
+$answer{'ADMIN_EMAIL'}          = 'vagrant@{{ WEB_HOSTNAME }}';
+$answer{'ADMIN_OK'}             = 'Y';
+$answer{'ADMIN_PASSWORD'}       = 'vagrant01!';
+$answer{'ADMIN_REALNAME'}       = 'Vagrant User';
+$answer{'NO_PAUSE'}             = 1;
+$answer{'apache_size_limit'}    = 700000;
+$answer{'bugzilla_version'}     = '1';
+$answer{'create_htaccess'}      = '';
+$answer{'db_check'}             = 1;
+$answer{'db_driver'}            = 'mysql';
+$answer{'db_host'}              = '{{DB_IP}}';
+$answer{'db_name'}              = 'bugs_bmo';
+$answer{'db_pass'}              = 'bugs';
```

Comment:
Same as above. I know it's just test data, but what if they load a sanitized copy into their DB and don't change it?